common/test/views/support/cleaner/AmpEmbedCleanerTest.scala

```diff
@@ -50,6 +49,33 @@ class AmpEmbedCleanerTest extends FlatSpec with Matchers {
     result.getElementsByTag("amp-soundcloud").size should be(0)
   }
 
+  "AmpEmbedCleaner" should "replace an iframe in an map element with an amp-iframe element" in {
+    val googleMapsUrl = "https://www.google.com/maps/embed/v1/place?center=-3.9834936%2C12.7024497&key=AIzaSyBctFF2JCjitURssT91Am-_ZWMzRaYBm4Q&zoom=5&q=Democratic+Republic+of+the+Congo"
+    val doc = s"""<html><body><figure class="element-map"><iframe src="$googleMapsUrl"></iframe></figure></body></html>"""
+    val document: Document = Jsoup.parse(doc)
+    val result: Document = clean(document)
+
+    result.getElementsByTag("amp-iframe").size should be(1)
+  }
+
+  "AmpEmbedCleaner" should "not add an amp-iframe element if an map element does not contain an iframe" in {
+    val doc = s"""<html><body><figure class="element-map"></figure></body></html>"""
+    val document: Document = Jsoup.parse(doc)
+    val result: Document = clean(document)
+
+    result.getElementsByTag("amp-iframe").size should be(0)
+  }
+
+  "AmpEmbedCleaner" should "create an amp-iframe element with an iframe from the iframe src" in {
+    val googleMapsUrl = "https://www.google.com/maps/embed/v1/place?center=-3.9834936%2C12.7024497&key=AIzaSyBctFF2JCjitURssT91Am-_ZWMzRaYBm4Q&zoom=5&q=Democratic+Republic+of+the+Congo"
```

Comment:
Are these all the same urls? Instead of defining them 3 times, could you make one `val` at the top of the file?